extensions/machine-learning-services/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "machineLearningServices",
```

Comment:
Is there value in having this extension in the ads source as opposed to a separate github repo?

Reply:
no, I just added here because other extensions are  added to ads repo. Is there any policy on what goes here and what doesn't? 

---
In reply to: [356828060](https://github.com/microsoft/azuredatastudio/pull/8622#discussion_r356828060) [](ancestors = 356828060)